src/main/java/frc/robot/Robot.java

```diff
@@ -108,6 +106,10 @@ public void robotPeriodic() {
         // and running subsystem periodic() methods.  This must be called from the robot's periodic
         // block in order for anything in the Command-based framework to work.
         CommandScheduler.getInstance().run();
+        SmartDashboard.putData(robotContainer.drivetrainSubsystem);
+        SmartDashboard.putData(robotContainer.armSubsystem);
+        SmartDashboard.putData(robotContainer.carriageSubsystem);
+        SmartDashboard.putData(robotContainer.elevatorSubsystem);
```

Comment:
I'm pretty sure this shouldn't be in the robotPeriodic. I think that putData() publishes the entry into the table and should only be called once. If this is the case, let's put it in robotInit()